source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,191 @@
+This will set up the parallel configuration.  
+
+::
+
+   $ matlab -nodisplay
+
+::
+
+   >> rehash toolbox;
+   >> configCluster;
+   
+You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. You have now created a cluster
+profile.
+
+Configuring additional properties
+*********************************
+
+It is possible to set some user-specific properties and save them in your cluster profile. You can always adapt these and save the profile again, without performing
+the previous step again. All of these properties are available in the ``AdditionalProperties`` property of your cluster profile. 
+   
+Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+
+::
+
+   >> c = parcluster;
+   
+``c = parcluster`` will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
+the correct profile will be chosen when using this command.
+
+If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+
+::
+
+   >> c = parcluster('local');
+
+You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
+other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+
+::
+
+   >> % View the additional properties
+   >> c.AdditionalProperties
+   >> % set the required properties
+   >> c.AdditionalProperties.AccountName = <account-name>;
+   >> c.AdditionalProperties.WallTime = '05:00:00';
+   >> % save the profile
+   >> c.saveProfile
+   
+Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+
+Submitting jobs
++++++++++++++++
+
+Submitting jobs will happen from within Matlab. There are two ways to do this. You can start a Matlab session on a login node, or you can first request an
+interactive job, where you launch your Matlab session. Which one you choose will depend on the workload you plan to have on the node before submitting a job.
+As for any other work on the cluster, you can always do minor work on the login node, but for heavier calculations (before submitting your multinode job), you should
+use an interactive session. 
+
+It is possible to use the Matlab GUI as well, on all login nodes. It is strongly recommended though to use NoMachine if you want to use the GUI though (login node
+3 and 4), as the starting the GUI from any other login node will be extremely slow. If you want to use a GUI on a compute node, you would need GPU resources
+(no X11 forwarding on the other nodes). Requesting GPU resources just for the GUI is quite expensive though. Once the Matlab Parallel Computing Toolbox is
```

Comment:
I don't think this is true (i.e. there is no need to request GPU resources in order to use the Matlab GUI in an interactive job on a compute node). Of course, to be sure that X forwarding is enabled one needs to use the `-X` qsub option. I checked this on a Skylake debug node and it worked fine with the `matlab/R2019a` module (both via jobs-submitted-via-NoMachine and via-MobaXterm). In general the NoMachine way should be more appropriate for users on a comparatively slow internet connection (in those cases the "plain-SSH-connection" way will be a bit laggy). Could you verify this with e.g. `MATLAB/R2022b` to see if this is module-dependent?

Reply:
Indeed, my bad. I had issues with the x forwarding in my previous tests for some reason. I am adding that now. On my side this is still quite laggy though, especially using anything 'clickable'